packages/matchbox/src/components/Tabs/Tabs.js

```diff
@@ -70,7 +70,7 @@ function Tab(props) {
 Tab.displayName = 'Tab';
 
 function Tabs(props) {
-  const { tabs, selected, onSelect, fitted, ...rest } = props;
+  const { disableResponsiveBehavior, tabs, selected, onSelect, fitted, ...rest } = props;
```

Comment:
When responsive behavior is disabled and the page is sized down the tabs overflow off the side of the page. Not sure if this will ever be an issue or not, but would it make sense to hide the overflow or even give it a scrollable overflow?

Reply:
Hmm I think hiding makes sense. Nick requested this so not sure what he is expecting